### DISCUSSION TITLE
500 Internal Server Error with Artifactory

### DISCUSSION THREAD

**Discussion by ohmkay** [iterate-ch/cyberduck#16429] [Q&A]
When connecting with WebDAV HTTPS to our local Artifactory instance, Cyberduck connects properly and most functionality works well (upload, download, delete etc).  However, when attempting to move or rename a file, Cyberduck throws a generic 500 Internal Server Error.

Artifactory does not provide any logs of the failed move or rename action.  However, the local Apache config does show the request:

```
IP_REMOVED  - - [09/Oct/2024:21:31:53 +0000] "MOVE /artifactory/cyberduck-test/test.txt HTTP/1.1" 500 95
```

What can I do to determine the cause of this 500 error?  I haven't been able to find any references of this issue with other users

**Comment by AliveDevil** [comment]
You can use curl to perform the request manually, and then get the whole picture of what's wrong - you can send us the Debug log from Cyberduck either here or via support@cyberduck.io, when you want us to provide the curl command for you.
A sample might look like:[^1]
[^1]: [RFC4918 Section 9.9.5](https://datatracker.ietf.org/doc/html/rfc4918#section-9.9.5)
```
curl -X MOVE --username "" "https://artifactory.domain.tld/artifactory/cyberduck-test/test.txt" -H "Destination: https://artifactory.domain.tld/artifactory/cyberduck-test/renamed.txt"
```

But, depending on your setup Apache might be unable to proxy the MOVE-verb to Artifactory.
